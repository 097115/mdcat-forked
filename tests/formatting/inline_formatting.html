<span id="line-0">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam <span style="font-weight: bold">nonumy</span></span>
<span id="line-1"><span style="font-weight: bold">eirmod tempor invidunt</span> ut labore et <span style="font-style: italic">dolore magna aliquyam erat</span>, sed diam</span>
<span id="line-2">voluptua. <span style="color: #aa5500">At vero eos et</span> accusam et</span>
<span id="line-3"></span>
