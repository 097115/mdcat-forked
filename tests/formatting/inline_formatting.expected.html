<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf8">
<title></title>
<style type="text/css">
.ansi2html-content { display: inline; white-space: pre-wrap; word-wrap: break-word; }
.body_foreground { color: #AAAAAA; }
.body_background { background-color: #000000; }
.body_foreground > .bold,.bold > .body_foreground, body.body_foreground > pre > .bold { color: #FFFFFF; font-weight: normal; }
.inv_foreground { color: #000000; }
.inv_background { background-color: #AAAAAA; }
.ansi1 { font-weight: bold; }
.ansi3 { font-style: italic; }
.ansi9 { text-decoration: line-through; }
.ansi33 { color: #aa5500; }
</style>
</head>
<body class="body_foreground body_background" style="font-size: normal;" >
<pre class="ansi2html-content">
<span id="line-0">Lorem <span class="ansi9">ipsum dolor sit amet</span>, consetetur sadipscing elitr, sed diam <span class="ansi1">nonumy</span></span>
<span id="line-1"><span class="ansi1">eirmod tempor invidunt</span> ut labore et <span class="ansi3">dolore magna aliquyam erat</span>, sed diam</span>
<span id="line-2">voluptua. <span class="ansi33">At vero eos et</span> accusam et</span>
<span id="line-3"></span>

</pre>
</body>

</html>
